Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift

```diff
@@ -70,16 +70,16 @@ public enum FeatureFlag: String, CaseIterable {
             false
         case .deselectChapters:
             false
+        case .newSettingsStorage:
+            true
```

Comment:
Isn't this the part that can cause more issues?

From what we've been seeing recently related to issues it seems the conversion is what is causing some issues, not necessarily the sync.

I think it would be safer to keep this disabled but enable it through the remote feature flag? 

Also, maybe we should enable this only on TestFlight/debug?

Reply:
> I think it would be safer to keep this disabled but enable it through the remote feature flag?

Ideally, I agree that we should use the remote feature flag but I don't think there's a good way for us to target only TestFlight builds outside of a code change right now.

> maybe we should enable this only on TestFlight/debug

I think it would be reasonable to keep `newSettingsStorage` and `settingsSync` the same here and only enable both for TestFlight. We can always change this prior to submitting an App Store release.

Reply:
> I think it would be safer to keep this disabled but enable it through the remote feature flag?

Another thing to note: It may be safer to keep these enabled at launch whenever possible because some settings may affect episode listings, notifications, up next, etc.